lib/Makefile

```diff
@@ -29,7 +29,7 @@ CFLAGS			+= -I. -DLTPLIB
 ifneq ($(ANDROID),1)
 FILTER_OUT_DIRS		+= android_libpthread android_librt
```

Comment:
This is a comment for previous commit (https://github.com/linux-test-project/ltp/commit/34a7c82a7a). I've asked on mailing list some time ago (http://lists.linux.it/pipermail/ltp/2017-March/003905.html).

I don't see directories lib/android_libpthread and lib/android_librt. Haven't you forgotten to add it to git or am I missing something?